backend/mbql/test/metabase/mbql/util_test.clj

```diff
@@ -1059,3 +1059,15 @@
                                 :query    {:source-query {:expressions  {:two [:+ 1 1]}
                                                           :source-table 1}}}
                                "two"))
+
+(expect
+  1
+  (mbql.u/field-clause->id-or-literal [:field-id 1]))
+
+(expect
+  "foo"
+  (mbql.u/field-clause->id-or-literal [:field-literal "foo" :type/Integer]))
+
+(expect
+  "foo"
+  (mbql.u/field-clause->id-or-literal [:expression "foo"]))
```

Comment:
@sbelak @robdaemon I just noticed this while doing a merge back into EE. Please don't write any new tests using `expect`. We're trying to convert everything over to `deftest` so this is just making more work for us. It takes the same amount of time to write with `deftest` anyway. Actually less because you could have done it like this:

```clj
(deftest field-clause->id-or-literal-test
  (doseq [[clause expected] {[:field-id 1] 1
                             [:field-literal "foo" :type/Integer] "foo"
                             [:expression "foo"] "foo"}
    (testing (pr-str clause)
      (is (= expected clause)))))
```